game-core/src/test/java/games/strategy/engine/message/RemoteActionCodeTest.java

```diff
@@ -14,33 +14,53 @@
 import org.junit.jupiter.params.aggregator.ArgumentsAggregator;
 import org.junit.jupiter.params.provider.CsvFileSource;
 
+/**
+ * Test class that verifies that remote operation codes didn't change unexpectedly due to some
+ * random change.
```

Comment:
nit, can probably omit "due to some random change", it's not 100% informative. You could perhaps give examples of what such changes would be, otherwise shorter and more information dense is perhaps better.

Reply:
I agree that "some random change" is kinda slang at this point.
What I wanted to express is that if you do some refactoring and mess with the interface structure, you might end up overriding another method unintentionally which could change the codes.
Or in case you change the codes on purpose, you need to consider updating the test cases as well.
Any suggestions on how to rephrase th sentence?

Reply:
```
 * Test class that verifies that remote operation codes didn't change unexpectedly. This can happen for a number of reasons, eg:
* - interface structure changed by overriding a method and changing the opcode that way
* - ...
```

> Or in case you change the codes on purpose, you need to consider updating the test cases as well.

That likely could be a  warning/comment on the annotation.